plugins/osdn/ovs/bin/openshift-sdn-ovs

```diff
@@ -76,6 +78,15 @@ add_ovs_flows() {
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=0, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=${tenant_id}, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
     fi
+
+    if [ -n "${ingress_bw}" ]; then
+        ovs-vsctl set port ${veth_host} qos=@newqos -- --id=@newqos create qos type=linux-htb other-config:max-rate=${ingress_bw}
+    fi
+
+    if [ -n "${egress_bw}" ]; then
+        ovs-vsctl set interface ${veth_host} ingress_policing_rate=${egress_bw}
+        ovs-vsctl set interface ${veth_host} ingress_policing_burst=500
```

Comment:
This blatantly looks like a cut+paste error and probably needs a comment clarifying that it isn't: "ingress" here means "ingress into ovs", and thus corresponds to the container's egress.

Also, why 500? And why hardcoded? The man page says "The burst size should be at least the size of the interface’s MTU" and also that "a value that is numerically at least as large as 10% of ingress_policing_rate helps TCP"